src/__tests__/Toast.spec.js

```diff
@@ -1,39 +1,38 @@
 import React from 'react';
-import { shallow, mount } from 'enzyme';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
 import { Toast } from '..';
 
 describe('Toast', () => {
   it('should render children', () => {
-    const toast = mount(<Toast>Yo!</Toast>);
-    expect(toast.text()).toBe('Yo!');
+    render(<Toast>Yo!</Toast>);
+    expect(screen.getByText('Yo!')).toBeVisible();
```

Comment:
We can use utility function `testForChildrenInComponent` from `src/utils.js` to test children. You can see an example in Form.spec.js

Reply:
Ah I see. I will correct it soon

Reply:
@illiteratewriter corrected. 
I also added a `testForCustomAttribute` in `testUtils.js`